Project_Mars/Project_Mars/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -1,29 +1,38 @@
-using SpecFlow_Mars.Pages;
-using SpecFlow_Mars.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+using System;
 
 
-namespace SpecFlow_Mars.StepDefinitions
+namespace Project_Mars.StepDefinitions
 {
     [Binding]
-    public class LanguageStepDefinitions : CommonDriver
+    [Parallelizable]
+    public class LanguageStepDefinitions 
     {
         LoginPage LoginPageobj = new LoginPage();
         HomePage HomePageobj = new HomePage();
         LanguageTab LanguageTabobj = new LanguageTab();
 
+        private IWebDriver driver;
 
+        public LanguageStepDefinitions(IWebDriver driver)
```

Comment:
how is the driver being initialised as before you had extend the class to common driver

Reply:
changed